source/common/upstream/subset_lb.cc

```diff
@@ -544,7 +544,11 @@ void SubsetLoadBalancer::HostSubsetImpl::update(const HostVector& hosts_added,
   }
 }
 
-HostSetImplPtr SubsetLoadBalancer::PrioritySubsetImpl::createHostSet(uint32_t priority) {
+HostSetImplPtr SubsetLoadBalancer::PrioritySubsetImpl::createHostSet(
+    uint32_t priority,
+    uint32_t over_provisioning_factor) {
+  (void)over_provisioning_factor;
```

Comment:
Is this normal Envoy style? I ask because I don't know and because the other option would be to leave the argument unnamed or named in a /* */ comment.

Reply:
UNREFERENCED_PARAMETER is the Envoy style. Thanks for pointing it out.